### ISSUE TITLE
loss.backward()时发生OOM现象

### ISSUE THREAD

**Issue by Algabeno** [LijunRio/Xrays_CT#12]
作者你好，我在使用V100训练自己的35个数据时，在loss.backward()时发生了OOM现象，这令我很疑惑，请问作者是怎么训练的？
![f65ead8b18591953202c84e2460b5e9](https://user-images.githubusercontent.com/66004210/200867243-845f4b01-1ac5-4b64-a87b-8bb1f59af835.jpg)

**Comment by Outstanding** [comment]
> 作者你好，我在使用V100训练自己的35个数据时，在loss.backward()时发生了OOM现象，这令我很疑惑，请问作者是怎么训练的？ ![f65ead8b18591953202c84e2460b5e9](https://user-images.githubusercontent.com/66004210/200867243-845f4b01-1ac5-4b64-a87b-8bb1f59af835.jpg)

您好，我也遇到了同样的问题，后来是怎么解决的呢？

**Comment by Hangiscoming** [comment]
> > 作者你好，我在使用V100训练自己的35个数据时，在loss.backward()时发生了OOM现象，这令我很疑惑，请问作者是怎么训练的？ ![f65ead8b18591953202c84e2460b5e9](https://user-images.githubusercontent.com/66004210/200867243-845f4b01-1ac5-4b64-a87b-8bb1f59af835.jpg)
> 
> 您好，我也遇到了同样的问题，后来是怎么解决的呢？
你好，请问该问题是怎样解决的呢？
